packages/apps/tools/src/utils/persist.ts

```diff
@@ -9,39 +9,10 @@ export const encode = (value: unknown) =>
 
 export const parse = (value: string) => JSON.parse(decodeURIComponent(value));
 
-export const setItem = (key: string, value: unknown) => {
+export const setItem = (key: string, value: unknown) =>
   Cookies.set(getName(key), encode(value));
-};
 
 export const getItem = (key: string) => {
-  try {
-    const cookie = Cookies.get(getName(key));
-    return cookie !== undefined ? parse(cookie) : undefined;
-  } catch (e) {
-    return undefined;
-  }
-};
-
-export const deleteItem = (key: string) => {
-  Cookies.remove(getName(key));
-};
-
-export const getItems = () => {
-  const cookies = Cookies.get();
-
-  return Object.keys(cookies).reduce(
-    (results, key) => {
-      const match: null | string[] = key.match(getName('(.*)'));
-      if (match) results[match[1]] = cookies[key];
-      return results;
-    },
-    {} as { [key: string]: string },
-  );
-};
-
-export const purge = () => {
-  const cookies = getItems();
-  Object.keys(cookies).forEach((key) => {
-    deleteItem(key);
-  });
+  const cookie = Cookies.get(getName(key));
```

Comment:
Why no longer is `try … catch`?

Reply:
trying out something, but will revert this